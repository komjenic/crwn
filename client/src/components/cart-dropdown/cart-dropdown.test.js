import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { CartDropdown } from './cart-dropdown.component';

it('expect to render CartDropdown component', () => {
    const cartItem = { cartItems: [] };
    const renderer = new ShallowRenderer();
    renderer.render(<CartDropdown {...cartItem} />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
});
