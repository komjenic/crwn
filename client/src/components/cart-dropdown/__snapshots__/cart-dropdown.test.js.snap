// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CartDropdown component 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.div)>
    <ForwardRef(styled.span)>
      Your cart is empty
    </ForwardRef(styled.span)>
  </ForwardRef(styled.div)>
  <ForwardRef(Styled(CustomButton))
    onClick={[Function]}
  >
    GO TO CHECKOUT
  </ForwardRef(Styled(CustomButton))>
</ForwardRef(styled.div)>
`;
